docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,178 @@
+Several properties are available to configure the behavior of this operator and they are summarized in the table below.
+
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *tablename* | Name of the table where data is to be inserted | String | Yes | N/A |
+| *fieldInfos*| JdbcFieldInfo maps a store column to a POJO field name| List | Yes | N/A |
+Those properties can be set like this:
+
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.prop.tablename</name>
+  <value>tablename</value>
+</property>
+
+<property>
+  <name>dt.operator.{OperatorName}.fieldInfosItem[0]</name>
+  <value>
+  {
+    "sqlType": 0,
+    "columnName":"ID",
+    "pojoFieldExpression": "id",
+    "type":"INTEGER"
+  }
+  </value>
+</property>
+
+<property>
+  <name>dt.operator.{OperatorName}.fieldInfosItem[1]</name>
+  <value>
+  {
+    "sqlType": 4,
+    "columnName":"NAME",
+    "pojoFieldExpression": "name",
+    "type":"STRING"
+  }
+  </value>
+</property>
+```
+
+
+## Platform Attributes that influence operator behavior
+
+| **Attribute** | **Description** | **Type** | **Mandatory** |
+| -------- | ----------- | ---- | ------------------ |
+| *input.TUPLE_CLASS* | TUPLE_CLASS attribute on input port which tells operator the class of POJO which is being received  | Class| Yes |
+Those properties can be set like this:
+
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.port.input.attr.TUPLE_CLASS</name>    
+  <value>com.example.mydtapp.PojoEvent</value>
+</property>
+```
+
+One of the concrete implementation is provided in Malhar as [JdbcPOJOInsertOutputOperator](https://github.com/apache/apex-malhar/blob/master/library/src/main/java/com/datatorrent/lib/db/jdbc/JdbcPOJOInsertOutputOperator.java).  The incoming tuples will be inserted in the table using PreparedStatement of the base class,  which is formed in activate() method of this operator.
+
+## Features
+The operator is **idempotent**, **fault-tolerant** and **statically partitionable**.
+
+## Partitioning of JDBC Output Operator
+#### Static Partitioning
+Only static partitioning is supported for this operator.
+This can be achieved in 2 ways
+
+1. Specifying the partitioner and number of partitions in the populateDAG() method
+```java
+JdbcPOJOInsertOutputOperator jdbcPOJOInsertOutputOperator = dag.addOperator("jdbcPOJOInsertOutputOperator", JdbcPOJOInsertOutputOperator.class);
+StatelessPartitioner<JdbcPOJOInsertOutputOperator> partitioner1 = new StatelessPartitioner<JdbcPOJOInsertOutputOperator>(2);
+dag.setAttribute(jdbcPOJOInsertOutputOperator, Context.OperatorContext.PARTITIONER, partitioner1);
+```
+
+2. Specifying the partitioner in properties file.
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.attr.PARTITIONER</name>
+  <value>com.datatorrent.common.partitioner.StatelessPartitioner:2</value>
+</property>
+```
```

Comment:
Changed the text in it, I am unable to figure out why this is happening hence avoiding the numbered list.